test/unit/logic/account.js

```diff
@@ -661,7 +661,7 @@ describe('account', () => {
 
 		it('should throw error when a numeric field receives non numeric value', done => {
 			account.merge(validAccount.address, { balance: 'Not a Number' }, err => {
-				expect(err).to.equal('Account#merge error');
+				expect(err).to.equal('Encountered insane number: NaN');
```

Comment:
any reason why this error message changed?

Reply:
Now the error is caught earlier than before.

Reply:
ok, in that case, shouldn't we cover the `Account#merge error` scenario with a different test?

Reply:
It is cover. I find at least 4 test cases.